examples/tracing/cache_miss.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python
+#
+# cache_miss.py Example of using BPF perf event.
+#               For Linux, uses BCC, eBPF. Embedded C.
+#
+# REQUIRES: Linux 4.7+ (BPF_PROG_TYPE_PERF_EVENT support).
+#
+# Copyright (c) 2016 Facebook, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License")
+#
+# 18-Oct-2016    Teng Qin    Created this.
+
+from __future__ import print_function
+import argparse
+from bcc import BPF, BPFPerfType, BPFPerfHWConfig
+import signal
+from time import sleep
+
+parser = argparse.ArgumentParser(
+    description="Summarize cache misses by PID or stack trace",
+    formatter_class=argparse.RawDescriptionHelpFormatter)
+parser.add_argument(
+    "--duration", default=10, type=int, help="Time to run, in seconds")
```

Comment:
I think duration would be the most commonly used option, in which case, I like the style of making the most commonly used option an optional positional parameter instead of an flag option. Like I did in profile.py. So "cachemiss" uses a default, and "cachemiss 5" runs for 5 seconds.